tests/manager_caller.c

```diff
@@ -12,8 +12,9 @@ main(int argc, char **argv)
     char *file = argv[1];
 
     struct Compartment *hw_comp = register_new_comp(file, true);
-    comp_map(hw_comp);
-    int comp_result = exec_comp(hw_comp, "main", NULL);
+    struct CompMapping *hw_map = mapping_new(hw_comp);
+    int comp_result = mapping_exec(hw_map, "main", NULL);
+    mapping_free(hw_map);
```

Comment:
Maybe a dumb question: can you do anything with a `hw_map` after `mapping_exec`? i.e. must you always follows `mapping_exec` with `mapping_free`?

Reply:
Yup, you can use it as many time as you want, with different inputs as well (currently not seen in tests) - this test [1] shows an (as of yet unported to the mapping system) example. I'll need to update this next, as I use this for benchmarking.

[1] https://github.com/capablevms/CHERI-ELF-comp/blob/master/tests/manager_caller_multiple.c#L22-L27